documentation/modules/rn-2.5.adoc

```diff
@@ -310,6 +304,12 @@ In {project-short} 2.5.6, the virt-v2v arguments include `–root first`, which
 
 In earlier releases of {project-short} {project-version}, populator pods were always restarted on failure. This made it difficult to gather the logs from the failed pods. In {project-short} 2.5.3, the number of restarts of populator pods is limited to three times. On the third and final time, the populator pod remains in the fail status and its logs can then be easily gathered by `must-gather` and by `forklift-controller` to know this step has failed. link:https://issues.redhat.com/browse/MTV-818[(MTV-818)]
 
+.Support deployment on OpenShift 4.15
+
+{project-short} 2.5.6 can be deployed on OpenShift 4.15 clusters. The operator includes a temporary fix for link:https://issues.redhat.com/browse/OCPBUGS-30095[Forklift operator fails on OpenShift 4.15]. The console plugin includes a fix for the `match` property that was removed in OpenShift Web Console. link:https://issues.redhat.com/browse/MTV-952[(MTV-952)]
```

Comment:
Do not forget to use attributes where possible

Either:
* :ocp: Red Hat OpenShift
OR
* :ocp-name: OpenShift

Reply:
Replaced all "OpenShift" with {ocp-name}